.github/workflows/shellcheck-image.yaml

```diff
@@ -39,5 +39,14 @@ jobs:
           registry: quay.io/redhat-pipeline-service
           username: ${{ secrets.QUAY_USERNAME }}
           password: ${{ secrets.QUAY_TOKEN }}
+      - name: Push to ghcr.io
+        id: push-to-ghcr
+        uses: redhat-actions/push-to-registry@v2
+        with:
+          image: ${{ steps.build-image.outputs.image }}
+          tags: ${{ steps.build-image.outputs.tags }} ${{ github.ref_name }}
+          registry: ghcr.io/${{ github.repository_owner }}/test
```

Comment:
Having  the images in different paths in different registries, defeat some of the purpose of having them in multiple registries. In the past using sub-paths for storing images was not supported. I am not sure if that is still the case with some of the registries. If it is I see two options:
- we say that we don't ship test images and as such we store them only in ghcr.io
- we create a second organization: ${{ github.repository_owner }}-test or something similar

In this respect having different names for the organizations in quay.io and ghcr.io defeats the purpose of having the images in multiple registries.

Reply:
How about I prefix test images with `test-` and leave them at the root of the registry?

Reply:
I implemented the change, let me know if you're not happy with the solution.

Reply:
I understand that it is not directly related to this PR but I am still concerned with 
ghcr.io/openshift-pipelines vs quay.io/redhat-pipeline-service
If users do `docker pull openshift-pipelines/cluster-setup:latest` and they don't have ghcr.io configured in their registry list best case they don't get the image, worst case they get someone's else image, possibly from an attacker.

Reply:
I thought that it is what we wanted to address by publishing to multiple registries.

Reply:
+1 to @fgiloux 's comment. I created this #174 a while back to track this.

Reply:
I also agree on the need to push to docker. That would be a separate PR though.